### ISSUE TITLE
The plugin of gridpanel is incorrectly serialized

### ISSUE THREAD

**Issue by zgcgwin** [extnet/Ext.NET#1774]
The C# code is :
```csharp
var cellEditing = new CellEditingPlugin()
{
       ClicksToEdit = 2,
};

Plugins = cellEditing;
```

and the render result is:
```js
plugins:"cellediting"
```

expected result is :

```js
plugins:[{ptype:"cellediting",clicksToEdit:2}]
```

**Comment by AndreyChechel** [comment]
Fixed in `7.0.0-preview4`.

The following markup

```cshtml
<ext-gridPanel>
    <plugin>
        <ext-cellEditingPlugin clicksToEdit="1" />
    </plugin>
</ext-gridPanel>
```
now generates the proper output including `ptype` property serialized:

```js
Ext.create("Ext.grid.Panel",{id:"ctl01",plugins:{ptype:"cellediting",clicksToEdit:1},renderTo:"App.ctl01_Container"});
```

Please pay attention, since `ClicksToEdit = 2` is the default value, it'll not be emitted.

**Comment by geoffreymcgill** [comment]
We have also confirmed the following scenario will render correctly.

```html
<ext-gridPanel>
    <plugin>
        <ext-cellEditingPlugin />
    </plugin>
</ext-gridPanel>
```
output:
```js
Ext.create("Ext.grid.Panel",{id:"ctl01",plugins:{ptype:"cellediting"}});
```
